### DISCUSSION TITLE
Unable to Build Playwright Image from Fresh Clone of Repo

### DISCUSSION THREAD

**Discussion by MichaelrMentele** [fastapi/full-stack-fastapi-template#1438] [Questions]
### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched in the documentation/README.
- [X] I already searched in Google "How to do X" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs/README and didn't find an answer.

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```Text
Fresh clone of template..?
```

### Description

From a fresh clone after manually updating the env variables, I am unable to run the `docker compose watch` command -- it fails to build the playwright image.

Two notes:
1. the Dockerfile.playwright fails to build and gets error `Failed to fetch http://deb.debian.org/debian/pool/main/f/fonts-ipafont/fonts-ipafont-gothic_00303-23_all.deb  Hash Sum mismatch`
2. when running `docker --debug compose watch` I see this warning `UndefinedVar: Usage of undefined variable '$VITE_API_URL' (line 13)` it seems the variable is undefined` but when I check the .env file it is set to `localhost`

### Operating System

macOS

### Operating System Details

Sequoia 15.2
Apple M2

### Python Version

Python 3.10.14

**Comment by MichaelrMentele** [comment]
I haven't tried to clear the cache yet, can also try on Ubuntu, I feel like Mac has lot's of random package failures with the new silicon still.

**Reply by YuriiMotov** [reply]
Have you tried it?
I just recently cloned the repo and everything is running without problems (ubuntu)
